spaghetto/src/Spago/Psa/Types.purs

```diff
@@ -0,0 +1,141 @@
+-- A majority of this code was copied from
+-- - https://github.com/natefaubion/purescript-psa-utils
+-- 
+-- To fullfil license requirements
+--   Copyright © Nathan Faubion
+--   https://opensource.org/license/mit/
+module Spago.Psa.Types
+  ( ErrorCode
+  , ModuleName
+  , Filename
+  , PsaOutputOptions
+  , PsaResult
+  , PsaError
+  , PsaAnnotedError
+  , PsaPath(..)
+  , Position
+  , Suggestion
+  , Lines
+  , psaResultCodec
+  , psaErrorCodec
+  , compareByLocation
+  ) where
+
+import Prelude
+
+import Data.Codec.Argonaut.Record as CAR
+import Data.Codec.Argonaut as CA
+import Data.Codec.Argonaut.Compat as CACompat
+import Data.Maybe (Maybe(..))
+import Data.Set (Set)
+import Data.Tuple (Tuple(..))
+import Spago.Core.Config as Core
+
+type ErrorCode = String
+type ModuleName = String
+type Filename = String
+type Lines = Array String
```

Comment:
I'm wondering if these type aliases should be dropped.

Reply:
I think they are generally useful, let's keep them.